hack/e2e.go

```diff
@@ -46,6 +46,7 @@ func parse(args []string) (flags, error) {
 	fs := flag.NewFlagSet(args[0], flag.ContinueOnError)
 	get := fs.Bool("get", getDefault, "go get -u kubetest if old or not installed")
 	old := fs.Duration("old", oldDefault, "Consider kubetest old if it exceeds this")
+	verbose := fs.Bool("-v", true, "this flag is translated to kubetest's --verbose")
```

Comment:
s/--verbose/--verbose-commands

Reply:
hmm had this locally, must not have pushed it. done.